pheps/phep-0003.md

```diff
@@ -0,0 +1,356 @@
+
+PyHC packages should clearly document their dependency version policy (e.g., like [PlasmaPy](https://docs.plasmapy.org/en/stable/contributing/coding_guide.html#python-and-dependency-version-support) and [SpacePy](https://spacepy.github.io/dep_versions.html)) and be tested against the minimum and maximum supported versions. Testing with CI against release candidates is encouraged, too, as a way to stay ahead of future releases. Packages that use semantic versioning should consider using their version number to indicate versions that drop support for older dependencies. There is no expectation that a package "deprecate" an older dependency before dropping support for it. However, there is an expectation that maximum or exact requirements (e.g., `numpy<2` or `matplotlib==3.5.3`) be set only when absolutely necessary (and that GitHub issues be immediately created to remove such requirements).
+
+This new policy will replace the current standard [#11](https://github.com/heliophysicsPy/standards/blob/main/standards.md#standards) in the PyHC standards document with the following new text:
+
+> **11. Python and Upstream Package Support:** All packages should support minor Python versions released within the last 36 months (3 years) and upstream core Scientific Python packages released within the last 24 months (2 years). Additionally, packages should support new versions within 6 months of their release (see [PHEP 3](https://github.com/heliophysicsPy/standards/pull/29)). 
+
+Lastly, if there is a Python 4 or other significant changes in dependencies, this policy will have to be reviewed in light of the community's and projects' best interests.
+
+# Backwards Compatibility
+<a name="backwards-compatibility"></a>
+This policy potentially introduces backwards incompatibilities by enforcing a new support timeline, which may encourage some packages to drop support for older dependency versions sooner than planned.
+
+# Security Implications
+<a name="security-implications"></a>
+There are no direct security implications of this policy. However, ensuring packages are updated to newer dependency versions may indirectly improve security by incorporating fixes and improvements from newer releases.
+
+# How to Teach This
+<a name="how-to-teach-this"></a>
+ - A new web page under the PyHC Projects page detailing the support schedule (similar to the Gantt chart in SPEC 0)? 
+ - Automated email reminders sent quarterly and/or near upcoming drop/support dates? 
+
+# Reference Implementation
+<a name="reference-implementation"></a>
+Multiple PyHC packages already follow this version support policy. One notable example is PlasmaPy which currently [documents their SPEC 0-based policy](https://docs.plasmapy.org/en/stable/contributing/coding_guide.html#python-and-dependency-version-support) and even mentions it in comments inside their [pyproject.toml](https://github.com/PlasmaPy/PlasmaPy/blob/main/pyproject.toml) file.
+
+## Code to generate support and drop schedules:
```

Comment:
Has this code been modified from the SPEC 0 version? if not can't we just link it?

Reply:
Yeah the code has been modified, to include 6-month "support by" dates.